code/WorkInProgress/Electronics.dm

```diff
@@ -783,7 +783,7 @@
 /obj/machinery/rkit/Topic(href, href_list)
 	if (usr.stat)
 		return
-	if ((in_interact_range(src, usr) && istype(src.loc, /turf)) || (issilicon(usr)))
+	if ((in_interact_range(src, usr) && istype(src.loc, /turf)) || (issilicon(usr)) || HasWirelessInteractor(usr))
```

Comment:
Isn't this already covered by `in_interact_range`?  I guess this covers instances when the player is inside something (closet/pod)... 🤷

Reply:
I was trying to fix an issue, this fix didn't work, forgot to uncommit
fuck